Dockerfile

```
@@ -0,0 +1,26 @@
+# Start with Python base image
+FROM python:3.9-slim
+
+# Set the working directory
+WORKDIR /app
+
+# Copy necessary files
+COPY . /app
```

```suggestion
COPY SimpleJadePinServer.py index.html oracle_qr.html qrcode.js /app/
```
Only copy the necessary files to the container.